source/Irrlicht/CNullDriver.cpp

```diff
@@ -192,6 +193,54 @@ void main()
         addBuiltInToCaches(sampler, "irr/builtin/samplers/default_clamp_to_border");
 	}
 
+    //images
+    core::smart_refctd_ptr<asset::ICPUImage> dummy1x1Image;
+    {
+        asset::ICPUImage::SCreationParams info;
+        info.format = asset::EF_R8_UNORM;
+        info.type = asset::ICPUImage::ET_2D;
+        info.extent.width = 1u;
+        info.extent.height = 1u;
+        info.extent.depth = 1u;
+        info.mipLevels = 1u;
+        info.arrayLayers = 1u;
+        info.samples = asset::ICPUImage::ESCF_1_BIT;
+        info.flags = static_cast<asset::IImage::E_CREATE_FLAGS>(0u);
```

Comment:
default color should be "oczojebny" pink/magenta with alpha 255

Hell, if you want to you can make it a 2x2 checkerboard of grey and magenta to be more obvious.